sdk/helper/keysutil/policy.go

```diff
@@ -818,7 +818,7 @@ func (p *Policy) convergentVersion(ver int) int {
 	return convergentVersion
 }
 
-func (p *Policy) Encrypt(ver int, context, nonce []byte, value string) (string, error) {
+func (p *Policy) Encrypt(ver int, context, nonce []byte, value string, paddingScheme string) (string, error) {
```

Comment:
I should double check I haven't violated this myself recently, but in general, we wish to avoid modifying these functions as they lie in our public `sdk/` space. 

I think you could change this to `Encrypt->EncryptWithAlgorithm` and have the former call the latter with `oaep` by default for consistency perhaps...? 

Then if/when we introduce ECEIS based schemes, I think it might make the most sense?

Reply:
No, I don't think our SDK has much documentation sadly. I think it'd be cool to get better inline godoc in the future, but we don't have [much presently](https://pkg.go.dev/github.com/hashicorp/vault/sdk/helper/keysutil). 

(If you want to work on this, happy to review such PRs in the future too!)